### DISCUSSION TITLE
webpack学习求组队

### DISCUSSION THREAD

**Discussion by xiumubai** [shengxinjing/it-roadmap#46] [找队友]
前端小菜一枚，最近刚入职开课吧，整理了一下我的博客，
博客链接：https://xiumubai.github.io/，
欢迎来撩

**Comment by xiumubai** [comment]
最近在整理webpack的学习资料，从0-1实现工程化，性能优化，源码分析，plugin和loader遍写等

**Comment by xiumubai** [comment]
博客每篇文章最下面的可以评论，使用github授权登陆即可，评论内容可以使用markdown的语法

**Comment by cooper1x** [comment]
页面404了，而且1L的链接后面多了个逗号。
